client/src/pages/StaffPage.css

```diff
@@ -1,17 +1,18 @@
 .staff-page{
-    width: 100%;
+    width: 100%-25rem;
```

Comment:
nitpick: i think in general when you do a calc like this its better to put it in a calc function. like
```
width: calc(100% - 25rem);
```
just to for safety when being used on different browsers.